### ISSUE TITLE
Error when registering a volume in a Nomad namespace

### ISSUE THREAD

**Issue by acornies** [hashicorp/terraform-provider-nomad#172]
### Terraform Version
Terraform v0.13.4
+ provider registry.terraform.io/hashicorp/nomad v1.4.10
+ provider registry.terraform.io/hashicorp/template v2.2.0
+ provider registry.terraform.io/terraform-provider-openstack/openstack v1.32.0

### Nomad Version
v1.0.0-beta2+ent

### Provider Configuration
Which values are you setting in the provider configuration? 

```hcl
provider "nomad" {
  address   = var.nomad_endpoint
  secret_id = var.nomad_token
}
```

### Environment Variables
None

### Affected Resource(s)
- nomad_volume

### Terraform Configuration Files
```hcl
resource "nomad_volume" "airflow_db" {
  type            = "csi"
  plugin_id       = data.nomad_plugin.cinder.plugin_id
  volume_id       = "airflow_db_pre"
  name            = "airflow_db_pre"
  external_id     = openstack_blockstorage_volume_v3.airflow_db.id
  access_mode     = "single-node-writer"
  attachment_mode = "file-system"
  namespace = var.namespace
}
```

### Debug Output
https://gist.github.com/acornies/4fd2691c358db25f47f2739607a79274

### Panic Output
N/A

### Expected Behavior
The Terraform Nomad provider should have registered a CSI volume in the correct namespace.

### Actual Behavior
The Terraform Nomad provider errors and registered a CSI volume in the default namespace.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
We're using nomad v1.0.0-beta2+ent (Enterprise) and the bug only happens when registering a volume using the namespace parameter.

### References
N/A

**Comment by lgfa29** [comment]
Hi @acornies, thanks for the report. This issue has been recently fixed in #169.

As a workaround until the next release is out, you can set the environment variable `NOMAD_NAMESPACE` to the namespace you would like to target before running `terraform apply`.

**Comment by acornies** [comment]
Thanks for the quick response and workaround!
